content/v1.0.0/index.md

```diff
@@ -7,41 +7,45 @@ aliases: ["/en/"]
 
 ## Summary
 
-The Conventional Commits specification is a lightweight convention on top of commit messages.
-It provides an easy set of rules for creating an explicit commit history;
-which makes it easier to write automated tools on top of.
-This convention dovetails with [SemVer](http://semver.org),
+The Conventional Commits specification is a simple set of rules for commit messages.
+It helps create an explicit commit history, and makes writing automation tools easier.
+This convention works well with [SemVer](http://semver.org),
 by describing the features, fixes, and breaking changes made in commit messages.
 
-The commit message should be structured as follows:
+The commit message should be structured as follows (`<>` denotes required
+parts, `[]` parts are optional):
 
 ---
 
 ```
-<type>[optional scope]: <description>
+<type>([scope])[!]: <description>
 
-[optional body]
+[body]
 
-[optional footer(s)]
+[footer(s)]
 ```
```

Comment:
I those add a lot of value, why are we removing them?

Reply:
Because by convention <> and [] already mark fields as required or optional. In this particular case it is not the specification that can be parsed or produced automatically, so the <> and [] can be removed if you prefer word descriptions.